tests/configs/base_config.yaml

```diff
@@ -18,14 +17,6 @@ hydra:
     # we change the working directory during the run/sweep directory
     # this is useful for saving outputs in a separate directory
     chdir: true
-  launcher:
-    # we set the number of jobs to 2 since when using 1, joblib reuses the same process
-    n_jobs: 2
-    prefer: processes
-    backend: multiprocessing
-  sweeper:
-    # now we force the sweeper to run one job at a time, achieving almost perfect isolation
-    max_batch_size: 1
```

Comment:
Why remove it in the end?

Reply:
I was getting some errors with ddp so I thought `joblib` might be interfering with `torch.multiprocessing`.
Turns out it was because the other backends were also imported (nncf) which altered the behavior of pytorch (used some unpicklable objects, specifically a lambda function). 
In the future it might be worth moving the model instantiation (for training benchlark) in the training worker to avoid any problems between pickle and modeling code.